cfgov/data_research/mortgage_utilities/s3_utils.py

```diff
@@ -13,7 +13,7 @@
 # bake_to_s3 functions require S3 secrets to be stored in the env
 BASE_BUCKET = settings.AWS_STORAGE_BUCKET_NAME
 MORTGAGE_SUB_BUCKET = "data/mortgage-performance"
-PUBLIC_ACCESS_BASE = 'https://{}/{}'.format(
+PUBLIC_ACCESS_BASE = 'https://s3.amazonaws.com/{}/{}'.format(
```

Comment:
This is necessary for this script to work with buckets other than files.consumerfinance.gov.